e2e/miniatures/test_themes_crud.py

```
@@ -0,0 +1,306 @@
+            # STEP 1: Navigate to Miniatures Themes tab
+            # ========================================
+            print("1. Navigating to Miniatures > Themes tab...")
+            page.goto(f"{BASE_URL}/miniatures")
+            page.wait_for_load_state("networkidle")
+            page.wait_for_timeout(500)
+
+            # Click Themes tab
+            themes_tab = page.locator("text=Themes").first
+            themes_tab.click()
+            page.wait_for_timeout(500)
+            page.screenshot(path="/tmp/themes_01_page.png")
```

Ruff (0.14.4)

Probable insecure usage of temporary file or directory: "/tmp/themes_01_page.png"

(S108)